packages/menu/README.md

```diff
@@ -173,6 +173,11 @@ import { Menu, MenuSeparator, MenuItem } from '@leafygreen-ui/menu';
 | `trigger`          | `function`, `React.ReactNode`            | A `ReactNode` against which the Menu will be positioned. The trigger prop can also support being passed a function. To work as expected, the function must accept an argument of `children`, which should be rendered inside of the function passed to trigger.         |              |
 | `usePortal`        | `boolean`                                | Will position Menu's children relative to its parent without using a Portal if `usePortal` is set to false. NOTE: The parent element should be CSS position relative, fixed, or absolute if using this option.                                                          | `true`       |
 | `adjustOnMutation` | `boolean`                                | Determines whether or not the `<Menu/>` should reposition itself based on changes to `trigger` or reference element position.                                                                                                                                           | `false`      |
+| `usePortal`        | `boolean`                                | Determines if the Menu will be rendered within a portal.                                                                                                                                                                                                                | `true`       |
+| `portalContainer`  | `HTMLElement` \| `null`                  | Sets the container used for the popover's portal.                                                                                                                                                                                                                       |              |
+| `scrollContainer`  | `HTMLElement` \| `null`                  | If the popover portal has a scrollable ancestor other than the window, this prop allows passing a reference to that lement to allow the portal to position properly.                                                                                                    |              |
+| `portalClassName`  | `string`                                 | Passes the given className to the popover's portal container if the default portal container is being used.                                                                                                                                                             |              |
+| `popoverZIndex`    | `number`                                 | Sets the z-index CSS property for the popover.                                                                                                                                                                                                                          |              |
 
 _Any other properties will be spread on the Menu `ul` container_
```

Comment:
did we move this into the prop table for most other components?

Reply:
Not sure – I didn't change it though.

Reply:
Can we update while we're in here?

Reply:
We have 3+ different ways we do this across our components. There isn't currently a standard that appears to be adopted by the majority of components.

I'm gonna leave this out for now since it seems better suited to a blanket update across our components (that we may or may not care about making at all in the existing README's if we're gonna eventually move the content entirely to .design).